src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -400,29 +400,29 @@ private static bool IsDriveRootedAbsolutePath(string path)
         /// <summary>
         /// Combine two paths, the first of which may be absolute.
         /// </summary>
-        /// <param name="rootOpt">First path: absolute, relative, or null.</param>
+        /// <param name="root">First path: absolute, relative, or null.</param>
         /// <param name="relativePath">Second path: relative and non-null.</param>
-        /// <returns>null, if <paramref name="rootOpt"/> is null; a combined path, otherwise.</returns>
+        /// <returns>null, if <paramref name="root"/> is null; a combined path, otherwise.</returns>
         /// <seealso cref="CombineAbsoluteAndRelativePaths"/>
-        public static string? CombinePossiblyRelativeAndRelativePaths(string? rootOpt, string? relativePath)
+        public static string? CombinePossiblyRelativeAndRelativePaths(string? root, string? relativePath)
```

Comment:
>string? root, string? relativePath [](start = 70, length = 34)

It looks like the only caller is `CombineAbsoluteAndRelativePaths`. If that's the case, consider changing the parameters to `string`.

Reply:
There are other callers in the IDE. I think it's ok to accept null root since we need to check for empty string anyways...